<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page2</class>
 <widget class="QWidget" name="Page2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>19</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>17</number>
         </property>
         <item>
          <widget class="QLabel" name="label_pay">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>73</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>73</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #f2f2f2;</string>
           </property>
           <property name="text">
            <string>PAY TO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_pay" native="true">
           <property name="minimumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #29333d;</string>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>461</width>
              <height>51</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="edit_pay_value">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>49</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>49</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #29333d;
color: #ffdfb5;</string>
               </property>
               <property name="frame">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_search_icon">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>29</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>29</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: transparent;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/icon/search.svg</normaloff>:/icon/search.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>17</number>
         </property>
         <item>
          <widget class="QLabel" name="label_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>73</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>73</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #f2f2f2;</string>
           </property>
           <property name="text">
            <string>LABEL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_label_value">
           <property name="minimumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #29333d;
color: #ffdfb5;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>17</number>
         </property>
         <item>
          <widget class="QLabel" name="label_amount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>73</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>73</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #f2f2f2;</string>
           </property>
           <property name="text">
            <string>AMOUNT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_amount_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>46</horstretch>
             <verstretch>49</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #29333d;
color: #ffdfb5;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_send">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #b6a27e;
color: #f2f2f2;</string>
           </property>
           <property name="text">
            <string>SEND</string>
           </property>
           <property name="autoRepeatDelay">
            <number>298</number>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>11</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_advance" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_swiftx">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #f2f2f2;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>SWIFTX</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="swiftx_toogle_layout"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>147</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
